src/Elcodi/Bundle/ProductBundle/composer.json

```diff
@@ -48,7 +48,6 @@
         "elcodi/media-bundle": "^1.0",
         "elcodi/currency-bundle": "^1.0",
         "elcodi/attribute-bundle": "^1.0",
-        "elcodi/configuration-bundle": "^1.0",
```

Comment:
Configuration and ConfigurationBundle are not used anymore.
Indeed, both packages are unsupported ;)